### PR TITLE
:sparkles: Allow ignoring moderation reports by subject

### PR DESCRIPTION
**Pull request by foysalit**
This PR adds a new filter to easily ignore moderation reports for certain subjects.

### CODE REVIEW

File: packages/pds/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/bluesky-social/atproto.git",
     "directory": "packages/pds"
   },
-  "main": "src/index.ts",
+  "main": "dist/index.js",
```
Comment (DavidBuchanan314) [inline packages/pds/package.json:10]: probably not an intentional change?

File: lexicons/com/atproto/admin/getModerationReports.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
         "type": "params",
         "properties": {
           "subject": {"type": "string"},
+          "ignoreSubjects": {"type": "array", "items": {"type": "string", "format": "did"}},
```
[the remark this answers is not in the document]
Comment (foysalit) [reply lexicons/com/atproto/admin/getModerationReports.json]: Agreed that it's not ideal to have that limit but like you said, for the use case right now, it should be fine.

**Comment by devinivy** [comment]
@foysalit looks good— if we can get this is implemented in the `bsky` package as well I think we'll be good to merge.

**Comment by foysalit** [comment]
@devinivy thanks for the reminder! ported over to bsky package. probably gonna need you to merge this though, since I don't have the permissions yet.

**Pull request merged**